ui/pages/Project/constants.js

```diff
@@ -262,11 +267,17 @@ const familyIsInReview = (family, individualsByGuid) => getFamilyCaseReviewStatu
 const familyIsAssignedToMe = (family, user) => (
   family.assignedAnalyst ? family.assignedAnalyst.email === user.email : null)
 
-const familyHasFeatures = (family, individualsByGuid) => {
+const REQUIRED_METADATA_FIELDS = INDIVIDUAL_DETAIL_FIELDS.filter(
+  ({ isRequiredInternal }) => isRequiredInternal,
+).map(({ field, subFields }) => (subFields ? subFields[0].field : field))
+
+const familyHasRequiredMetadata = (family, individualsByGuid) => {
   const individuals = family.individualGuids.map(
     individualGuid => individualsByGuid[individualGuid],
   ).filter(individual => individual)
-  return individuals.length ? individuals.some(({ features }) => (features || []).length > 0) : family.hasFeatures
+  return individuals.length ? individuals.some(individual => REQUIRED_METADATA_FIELDS.every(
+    field => individual[field] || individual[field] === false,
+  ) && individual.features.length > 0) : family.hasRequiredMetadata
```

Comment:
Is the `individual.features` still possible to be null?

Reply:
no because `features` is included in `REQUIRED_METADATA_FIELDS` and we just ran the `REQUIRED_METADATA_FIELDS.every(field => individual[field])`